### ISSUE TITLE
Upgrade to CosmWasm v0.14.0

### ISSUE THREAD

**Issue by ethanfrey** [CosmWasm/cosmwasm-plus#229]

**Comment by maurolacy** [comment]
This looks like a duplicate of #225. Based on sequence / priorities, I think it makes sense to work on this now, right? I can do it.

**Comment by ethanfrey** [comment]
What short memory I have... 6 days later making the same issue. I will close the other one.

As to priorities. Once you feel CosmWasm work is "done" (no planned api changes for v0.14.0, just waiting on feedback if there are bugs), then this would be great to start (as it unblocks some other work).

**Comment by maurolacy** [comment]
> Once you feel CosmWasm work is "done" (no planned api changes for v0.14.0, just waiting on feedback if there are bugs), then this would be great to start (as it unblocks some other work).

I think we can start migrating this, and revisit if needed.
